packages/css/src/highlight/highlight.scss

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+.amsterdam-highlight {
+  padding-block: var(--amsterdam-highlight-padding-block);
+  width: 100%;
```

Comment:
Why is this useful?

```suggestion
```

Reply:
Mostly an old habit to make sure it fills the container. But we can do without.